GamingAnthems/dist/metadata.json

```diff
@@ -13,15 +13,15 @@
     "raidstop.com"
   ],
   "version": "1.0.0",
-  "logo": "https://cdn.discordapp.com/attachments/681603738468614247/692449497921355846/logo_blacvk.png",
-  "thumbnail": "https://cdn.discordapp.com/attachments/681603738468614247/692449501197369404/logo_pink_and_black.png",
+  "logo": "https://imgur.com/9PEupja",
+  "thumbnail": "https://imgur.com/j7u9H9X",

```

Comment:
you need the i.imgur links and ending with .png / .jpg

Reply:
https://user-images.githubusercontent.com/51294309/77699256-fcd20580-6fa9-11ea-93cc-1d0584d821bc.png

Reply:
it works

Reply:
but they arent the direct image links, you need those

Reply:
done